ra/ra.go

```diff
@@ -778,24 +778,30 @@ func (ra *RegistrationAuthorityImpl) checkLimits(ctx context.Context, names []st
 }
 
 // UpdateRegistration updates an existing Registration with new values.
-func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (reg core.Registration, err error) {
-	base.MergeUpdate(update)
+func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (core.Registration, error) {
+	changed := base.MergeUpdate(update)
```

Comment:
This can be part of the `if`:

`if changed := base.MergeUpdate(update); !changed {`